src/dir.rs

```diff
@@ -200,20 +200,33 @@ impl<'a, IO: ReadWriteSeek, TP: TimeProvider, OCC: OemCpConverter> Dir<'a, IO, T
     ///
     /// `path` is a '/' separated directory path relative to self directory.
     ///
+    /// An empty path returns the current directory.
```

Comment:
I'm not sure this is a good way to handle this situation. Do you know any other API that behaves like this? What problem does this change solve?